src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/not-seeing-control-plane-data.mdx

```diff
@@ -37,7 +37,7 @@ You have completed the [installation procedure](/docs/kubernetes-monitoring-inte
       time="2022-06-21T12:21:25Z" level=debug msg="No \"scheduler\" pod has been discovered"
       ``` 
 
-      In this case you can change the discovery behavior with the `controlplane.config.[component].autodiscover[].selector` config of the helm [chart values](https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/values.yaml). Read more about [control plane components](/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3/#nrk8s-controlplane).
+      In this case you can change the discovery behavior with the `controlplane.config.[component].autodiscover[].selector` config of the helm [chart values](https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/values.yaml). Read more about [control plane components](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3/#nrk8s-controlplane).
```

Comment:
```suggestion
      In this case you can change the discovery behavior with the `controlplane.config.[component].autodiscover[].selector` config of the helm [chart values](https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/values.yaml). [Read more about control plane components](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3/#nrk8s-controlplane).
```

Reply:
I've changed the url, but I'm going to leave the `[` where it was, if you don't mind. The link refers to the control plane components.